### ISSUE TITLE
Deposit transaction is not confirmed

### ISSUE THREAD

**Issue by amalcs23** [stellar/django-polaris#296]
Hi Team,
I was able to create anchor in my localhost. But at the time of deposit  the transaction status is still showing **pending_user_transfer_start**. 

Terminal Output
server  | 2020-08-26 09:13:38,574 - INFO - "GET /sep24/transaction?id=d7d96793-9447-4ff8-82e8-f1f3b072540a HTTP/1.1" 200 656

**Comment by JakeUrban** [comment]
Hi @amalcs23 ,

There could be a few things happening that makes the transaction stay in `pending_uesr_transfer_start`. Lets start from the top,
- Do you have a `poll_pending_deposits` process running alongside the server?
    - ex. `python manage.py poll_pending_deposits --loop`
    - This process queries for `pending_user_transfer_start` transactions and passes them to `RailsIntegration.poll_pending_deposits()`
- `RailsIntegration.poll_pending_deposits()` must be implemented to return the pending deposits that are ready to be submitted to the Stellar network
    - On testnet, every transaction should be returned because real deposit funds are not used
- If you have both the integration implemented and the CLI process running, you should see logs from `poll_pending_deposits`, not just `server`

Let me know if that helps.

**Comment by amalcs23** [comment]
Hi @JakeUrban 
Thank you for your response.
I just clone the django-polaries repo and try to run the example. Is there any additional configuration needed?
I Made the changes mentioned in the readme file.

**Comment by JakeUrban** [comment]
@amalcs23 your `Asset` doesn't have a `distribution_seed` value, so `distribution_account` is `None` since it is derived from `distribution_seed`. Have you issued your asset on testnet?

**Comment by amalcs23** [comment]
Hi @JakeUrban 
I am creating asset using **http://localhost:8000/admin** .
In this UI form to create new asset there is no field for **distribution_seed**.
Is there any other way to create a new asset?

**Comment by JakeUrban** [comment]
Hi @amalcs23 , looks like your SEP-24 service is working! Good job figuring out how to add the asset using the CLI. The problem now seems to be related to the distribution account saved to the DB.

`op_underfunded` means the account used as the `sourceAccount` for the payment op doesn't have enough units of the asset you're referencing using the asset code & issuer public key.

This means one of two things:
- Your distribution account doesn't have enough of the issued asset for the requested deposit, or
- The asset code & issuer pair _or_ the distribution account referenced by the anchor is incorrect

Make sure your asset's issuer and distribution account are properly setup, potentially using [this tool](https://github.com/stellar/create-stellar-token) my coworker built. Then make sure your `Asset.distribution_seed` and `issuer` columns reference the keypairs used with the token issuer tool.

**Comment by JakeUrban** [comment]
It is strange that the transaction reached a `completed` status even though the deposit transaction failed though. Do you know how this may have happened? Do you update the transaction status in your integration functions?

**Comment by amalcs23** [comment]
Hi @JakeUrban 
The tool that you mentioned is not working. 
1. I clone the repo and run the command **node indes.js**, Then I got the error
> Seed: SD43UST4RQMBE7HWFXWISELVPBXNVTDV2GHQNFOHPNJXBOSUJTJVRNIK
> Pub : GCK3LF5MS3WFTLNGGMNRIQ6FV6OTBPOWJCOTUYH4PIKR6HLJ2ZS4RWFV
(node:13854) UnhandledPromiseRejectionWarning: Error: connect ETIMEDOUT 52.205.179.221:443
    at node_modules/stellar-sdk/lib/call_builder.js:245:47
    at step (
(node:13854) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:13854) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

2. When run the command **npx create-stellar-token** I get the error
npm ERR! cb.apply is not a function
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/amal/.npm/_logs/2020-09-01T08_13_01_139Z-debug.log
Install for [ 'create-stellar-token@latest' ] failed with code 1

Looking for suggetions

**Comment by amalcs23** [comment]
Hi @JakeUrban 
1.I was able to create my-own token and which is  successfully transferred to another account. I was able to verify the results in **https://testnet.steexp.com/**. But in **http://sep24.stellar.org**/, the newly created token is not shown. It still showing pending in mobile screen. But in terminal status is postMessage: Interactive webapp completed. 

2. Once the usser click on the deposit button it automatically deposit the requested amount to the wallet. Then how to implement anchor's approval. Other wise anyone can request any amount of tokens, which will automatically recorded to their wallet.
